src/tools/linkchecker/main.rs

```diff
@@ -142,6 +142,12 @@ fn is_exception(file: &Path, link: &str) -> bool {
     if let Some(entry) = LINKCHECK_EXCEPTIONS.iter().find(|&(f, _)| file.ends_with(f)) {
         entry.1.contains(&link)
     } else {
+        // FIXME(#63351): Concat trait in alloc/slice reexported in primitive page
+        if file.ends_with("std/primitive.slice.html") {
+            if link.ends_with("std/primitive.slice.html") {
+                return true;
```

Comment:
Can you put this in LINKCHECK_EXCEPTIONS instead of special-casing it?

Reply:
It cannot because the resolved path is outside build/doc dir, so the strip_prefix method in line 283 doesn't work.

Reply:
Got it, thanks. Can you add that as a comment?

Reply:
added it. But I'm not sure my words are clear enough.